pixeltable-mcp/base-sdk/tools.py

```
@@ -0,0 +1,273 @@
+
+    Example:
+        add_computed_column("my_table", "full_name", "table.first_name + ' ' + table.last_name")
+        add_computed_column("my_table", "yoy_change", "table.pop_2023 - table.pop_2022")
+    """
+    try:
+        table = pxt.get_table(table_name)
+        if table is None:
+            return f"Error: Table {table_name} not found."
+
+        # Replace 'table.' with the actual table reference
+        modified_expr = expression.replace("table.", f"table.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)